### PR TITLE
[TM-4913] Manage Entry Level Positions

### PR DESCRIPTION
**Pull request by jlam-mpc**

### CODE REVIEW

**Comment by pveland621** [comment]
looks like the clear filter button is getting the grey background highlight as well when you're in edit mode:
image

**Comment by pveland621** [comment]
Does this need to be MC `End` Date?
image

**Comment by pveland621** [comment]
bug editing two cards at once
to recreate:

1. click edit on two separate position cards
2. click a few checkboxes on the first card (LNA or FICA, etc.)
3. go to click the same checkboxes on the second card - they will check and uncheck checkboxes from the first card 

image

**Comment by scott062** [comment]
> bug editing cards to recreate:
> 
> 1. click edit on two separate position cards
> 2. click a few checkboxes on the first card (LNA or FICA, etc.)
> 3. go to click the same checkboxes on the second card - they will check and uncheck checkboxes from the first card
> 
> same thing when editing 3+ cards, they all only modify the first card to be selected for editing
> 
> EDIT: this is actually happening regardless of the number of cards you're editing - user can only edit the first card's checkboxes, checking boxes on any other cards doesn't work and that instead checks/unchecks boxes from the first card
> 
> image

This is probably bc of the `id` field on inputs - just need to generate dynamic ids to avoid this issue

File: src/Components/AdministratorPage/ManageEntryLevel/ManageEntryLevel.jsx [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+    selectedOrgs,
+    selectedGrades,
+    selectedSkills,
+    selectedJobs,
+    selectedLanguages,
+    overseas,
+    domestic,
+  ]);
+
+  function renderSelectionList({ items, selected, ...rest }) {
```
Comment (elizabeth-jimenez) [inline src/Components/AdministratorPage/ManageEntryLevel/ManageEntryLevel.jsx:142]: should be able to replace with utilities one

File: src/Components/AdministratorPage/ManageEntryLevel/ManageEntryLevel.jsx [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+                  onClick={resetFilters}
+                  disabled={disableSearch}
+                >
+                  <FA name="times" />
+                  Clear Filters
+                </button>
+              }
+            </div>
+          </div>
+          <div className="usa-width-one-whole position-search--filters--pv-man results-dropdown">
```
Comment (elizabeth-jimenez) [inline src/Components/AdministratorPage/ManageEntryLevel/ManageEntryLevel.jsx:208]: `position-search--filters--pv-man` this is the projected vacancy filter styling, we'll want to either create one for Manage EL or make some other one more generic in naming and reuse it (but not pv-man, cause 'Job Categories' is on two lines, so needs more space)

File: src/Components/AdministratorPage/ManageEntryLevel/EntryLevelCard.jsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+      { 'Location': getPostName(get(pos, 'post') || NO_POST) },
+      { 'Org/Code': getResult(pos, 'bureau_code') || NO_ORG },
+      { 'Grade': getResult(pos, 'grade') || NO_GRADE },
+      { 'Job Category': 'None Listed' },
+      { '': <CheckBox id="el" label="EL" value={el} disabled /> },
+      { '': <CheckBox id="lna" label="LNA" value={lna} disabled /> },
+      { '': <CheckBox id="fica" label="FICA" value={fica} disabled /> },
+      { '': <CheckBox id="mc" label="MC" value={mc} disabled /> },
+      { 'MC Date': getResult(pos, 'mc_date') || '---' },
+    ],
```
Comment (elizabeth-jimenez) [inline src/Components/AdministratorPage/ManageEntryLevel/EntryLevelCard.jsx:71]: missing `bodySecondary` with: Language
- O/D (Overseas, Domestic)
- Incumbent 
- Incumbent TED
- Assignee
- Assignee TED

**Pull request merged**